src/index.html

```diff
@@ -16,6 +16,7 @@
       sizes="128x128"
       href="/images/app_icon_128.png"
     />
+    %%PLACEHOLDER_RUM%%
```

Comment:
Currently this string shows up as readable content during development, but it's no longer visible as soon as the actuall app content appears. Any idea how to prevent this? Shall I make it a fake HTML tag?

Reply:
It seems https://en.wikipedia.org/wiki/Conditional_comment are spared from removal. So maybe try `<!--[if true]>PLACEHOLDER_RUM<![endif]-->` or something along the line

Reply:
I settled for `<placeholder_rum/>`. Uppercase didn't work, because it would be transformed to lowercase during `yarn dist`.